src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,26 @@
+export const ArticleCard = (props) => {
+    console.log(props.article.article_image_url)
```

Comment:
Don't leave console logs in, especially as this is logging for EVERY article in the database